usedapp/src/config/rootstock.js

```diff
@@ -0,0 +1,57 @@
+import { getAddressLink, getTransactionLink } from '../utils';
+
+// Rootstock networks configuration
+// List of other available networks:
+// https://github.com/TrueFiEng/useDApp/tree/master/packages/core/src/model/chain
+
+const rootstockTestnetExplorerUrl = 'https://explorer.testnet.rsk.co/';
+
+export const RootstockTestnet = {
+  chainId: 31,
+  chainName: 'Rootstock Testnet',
+  isTestChain: true,
+  isLocalChain: false,
+  rpcUrl: 'https://public-node.testnet.rsk.co',
+  multicallAddress: '0xcA11bde05977b3631167028862bE2a173976CA11',
+  nativeCurrency: {
+    name: 'Test Rootstock Bitcoin',
+    symbol: 'tRBTC',
+    decimals: 18,
+  },
+  getExplorerAddressLink: getAddressLink(rootstockTestnetExplorerUrl),
+  getExplorerTransactionLink: getTransactionLink(rootstockTestnetExplorerUrl),
+};
+
+const rootstockMainnetExplorerUrl = 'https://explorer.rsk.co/';
+
+export const RootstockMainnet = {
+  chainId: 30,
+  chainName: 'Rootstock Mainnet',
+  isTestChain: false,
+  isLocalChain: false,
+  rpcUrl: 'https://public-node.rsk.co',
+  multicallAddress: '0xcA11bde05977b3631167028862bE2a173976CA11',
```

Comment:
same here. pls add a comment about which multicall implementation this is (e.g. link to its github repo) ... additionally perhaps this is worth mentioning in the README.